spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -20,29 +20,29 @@ Although a simple concept, an `ItemReader` is
 the means for providing data from many different types of input. The most
 general examples include:
 
-* Flat File- Flat File Item Readers read lines of data from a
-flat file that typically describe records with fields of data
+* Flat File: Flat File Item Readers read lines of data from a
```

Comment:
They'd do better as "Flat-file item readers". There should be a hyphen in there, too. I was thinking that it might be called that all the way through the document for some reason, so I left it as it was. But it should be "Flat-file item readers".